packages/address/src/index.ts

```diff
@@ -1,3 +1,6 @@
+import * as testUtilities from './testUtilities';
```

Comment:
So the problem with adding this to your index file is that it will be added to the main bundle of your app, which might have some weird side effects if you import any jest related stuff, and will add to the size.

Reply:
Hmm, I wasn't sure about this one. I thought it would be ok as long as we don't do imports other than in our test files.